src/graphql/resolveProcedure.js

```diff
@@ -34,13 +39,26 @@ const resolveProcedure = (procedure, getProcedureArgs) => {
   return async (source, args, { client }) => {
     const procedureArgs = getProcedureArgs(source, args)
 
-    // Actuall run the procedure using our arguments.
-    const result = await client.queryAsync({
-      name: query.name,
-      text: query.text,
-      values: argEntries.map(([name]) => procedureArgs[camelCase(name)]),
+    const values = argEntries.map(([name, type]) => {
+      const obj = procedureArgs[camelCase(name)]
+      let value
+
+      if (type.isTableType) {
+        value = reduce(obj, (acc, val, key) => {
+          acc[key] = convertToPostgresArrayIfArray(val)
+          return acc
+        }, {})
+      }
+      else {
+        value = convertToPostgresArrayIfArray(obj)
```

Comment:
No it doesnt. Honestly I expected it to break. I am curious as why it doesn’t. For example shouldn’t it break when values has the following form `['c', true, ["a", "b", "c"]]`.